src/interactions/commands/mp3.ts

```diff
@@ -31,12 +60,14 @@ export default {
 
         await interaction.editReply({
             content: `${Emote.success} Here's your audio file!\n${Config.ad}`,
-            files: [res]
+            files: [{
+                attachment: res,
+                name: 'audio.mp3'
```

Comment:
I like time more because each time a user download's it, it won't append `(1)` to the file, plus if a user sorts the explorer after "name", it still will be sorted after creation date